ExtraEnemyCustomization/Customizations/Abilities/Handlers/HealthRegenHandler.cs

```diff
@@ -15,14 +16,14 @@ public class HealthRegenHandler : MonoBehaviour
 
         public HealthRegenData RegenData;
 
-        private float _RegenInitialTimer = 0.0f;
-        private float _RegenIntervalTimer = 0.0f;
-        private bool _IsRegening = false;
-        private bool _IsInitialTimerDone = false;
-        private bool _AlwaysRegen = false;
-        private bool _IsDecay = false;
-        private float _RegenCapAbsValue = 0.0f;
-        private float _RegenAmountAbsValue = 0.0f;
+        private float _regenInitialTimer = 0.0f;
+        private float _regenIntervalTimer = 0.0f;
+        private bool _isRegening = false;
+        private bool _isInitialTimerDone = false;
+        private bool _alwaysRegen = false;
+        private bool _isDecay = false;
+        private float _regenCapAbsValue = 0.0f;
+        private float _regenAmountAbsValue = 0.0f;
 
         private Action<EnemyAgent, Agent> _OnDamageDel;
```

Comment:
Inconsistent naming with the rest of the members
```suggestion
        private Action<EnemyAgent, Agent> _onDamageDel;
```